documentation/modules/exploit/multi/misc/weblogic_deserialize_badattrval.md

```diff
@@ -0,0 +1,102 @@
+## Vulnerable Application
+
+  There exists an object deserialization vulnerability
+  in multiple versions of WebLogic.
+
+  Unauthenticated remote code execution can be achieved
+  by sending a serialized BadAttributeValueExpException object
+  over T3 to vulnerable WebLogic servers.
+
+  This module has been tested against versions `v12.1.3.0.0`,
+  `v12.2.1.3.0`, and `v12.2.1.4.0`.
+
+  WebLogic versions can be downloaded from [here](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html).
+  
+### Installation
+
+  Some version of Java 8 jdk is required to be installed on the server.
+  This module has been tested successfully using jdk8u202 and [jdk8u251](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html).
+  
+  Installation instructions for WebLogic can be found [here](https://docs.oracle.com/cd/E24705_01/doc.91/e21052/appx_install_wls.htm#EOPWC376).
+  
+  On step 10 of the installation instructions keep the
+  `Run Quickstart` box checked and click `done`. A new window
+  should pop up. Select `Create a new domain` -> `next`.
+  Ensure `Basic WebLogic Server Domain` is selected and click `next`.
+  Create credentials and select `next`. Domain mode can be either
+  `Production` or `Development`, click `next`. Click `next` again
+  and select `Create`. Click `next` a couple more times, then click
+  `finish`.
+
+  To start WebLogic, execute either the `startWebLogic` script in
+  `Oracle/Middleware/Oracle_Home/user_projects/domains/base_domain/`.
```

Comment:
Either...?

Reply:
Ooops, I had originally put both the Linux script path and Windows script path in that section.